common/flatpak-installation.c

```diff
@@ -1440,6 +1440,38 @@ flatpak_installation_get_config (FlatpakInstallation *self,
   return flatpak_dir_get_config (dir, key, error);
 }
 
+/**
+ * flatpak_installation_get_min_free_space_bytes:
```

Comment:
Needs to be listed in `doc/reference/flatpak-sections.txt`. It’ll also need to be added to `debian/libflatpak0.symbols` in the `debian-master` branch (as a separate PR).